Tests/MongoSwiftTests/APMTests.swift

```diff
@@ -0,0 +1,191 @@
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+import MongoSwift
+import Nimble
+import NIO
+import NIOConcurrencyHelpers
+import TestsCommon
+
+@available(macOS 10.15, *)
+
+private protocol StreamableEvent {
+    var type: EventType { get }
+}
+
+extension CommandEvent: StreamableEvent {}
+
+extension SDAMEvent: StreamableEvent {}
+
+final class APMTests: MongoSwiftTestCase {
```

Comment:
I think we'll need the `@available` annotation here for these tests to build on macOS.